types/FBShareButton.d.ts

```diff
@@ -1,6 +1,3 @@
-/* eslint-disable @typescript-eslint/no-use-before-define */
```

Comment:
Didn't your linter complain about `styles` being used before it's declared?

Reply:
Yes but this actually a typescript issue and not a type declarations error